src/main/resources/public/js/index.js

```diff
@@ -0,0 +1,84 @@
+
+    /**
+     * Obtiene una pregunta
+     * @param {number} questionID es el numero de pregunta
+     */
+    function getQuestionByID(questionID) {
+        return new Promise(function(resolve, reject) {
+            $.get(`/question/get?idQuestion=${questionID}`, (questionJSON) => {
```

Comment:
La forma en la que escribiste el _endpoint_ de la API (`/question/get?idQuestion=${questionID}`) no se parece a los ejemplos que vimos en donde se describen buenas practicas.

[Aquí](https://hackernoon.com/restful-api-designing-guidelines-the-best-practices-60e1d954e7c9) y [aquí](https://blog.mwaysolutions.com/2014/06/05/10-best-practices-for-better-restful-api/) encontramos que el recurso debe estar escrito en plural y seguido de una diagoan y el _id_ de lo que buscas. Por ejemplo, siguiendo esto el _endpoint_ puede quedar de la siguiente forma: `/question/${questionID}`